docs/buildtest_perlmutter.rst

```diff
@@ -112,5 +159,8 @@ First, let's build this test and analyze the output::
   buildtest build -b perlmutter_tutorial/ex5/stream.yml
   buildtest inspect query -o stream_test
 
-Take a close look at the metrics value. In this task, you are requested to use use :ref:`assert_ge` with metric ``copy`` and
-``scale`` with reference value. For reference value please experiment with different metrics and see if test pass/fail.
+**TODO**
+
+- Check the output of metrics ``copy`` and ``scale`` in the command **buildtest inspect query -o stream_test**
+- Use :ref:`assert_ge` check with metric ``copy`` and ``scale`` and specify a reference value. Specify a reference value (pick some high number) that will make test fail and rerun test.
```

Comment:
Use the assert_ge check with metrics...

You mention specifying a reference value twice, that should probably be cleaned up